components/search/search.tsx

```diff
@@ -222,12 +228,47 @@ const Search: React.FC<Props> = (props: Props): JSX.Element => {
             actions.updateRhsState(RHSStates.SEARCH);
         }
         actions.updateSearchTerms('');
+        actions.updateSearchType('');
     };
 
     const handleShrink = (): void => {
         props.actions.setRhsExpanded(false);
     };
 
+    const handleSetSearchFilter = (filterType: SearchFilterType): void => {
+        switch (filterType) {
+        case 'documents':
```

Comment:
Is there, by chance, an existing mime type library that we could use to manage this mapping?

Reply:
This is a mapping between our set of filter types and file extensions, we could find something more "generic" but I think that is good enough for now.